lib/main_common.pm

```diff
@@ -1375,8 +1375,6 @@ sub load_extra_tests_desktop {
     }
     elsif (check_var('DISTRI', 'opensuse')) {
         if (gnomestep_is_applicable()) {
-            # Setup env for x11 regression tests
-            loadtest "x11/x11_setup";
```

Comment:
You don't bring this back, so we don't need it anymore?

Reply:
I think consoletest_setup include all test in x11_setup. So , we don't need it once consoletest_setup used in create_hdd_*